spec/config.spec.ts

```diff
@@ -24,104 +24,127 @@ import * as mockRequire from 'mock-require';
 import { expect } from 'chai';
 import { config, firebaseConfig } from '../src/config';
 
-describe('config()', () => {
+const NODE8_NODE10_PATH = '/srv/.runtimeconfig.json';
+const NODE6_PATH = '/user_code/.runtimeconfig.json';
+
+describe.only('config()', () => {
   afterEach(() => {
     mockRequire.stopAll();
     delete config.singleton;
     delete process.env.FIREBASE_CONFIG;
-    delete process.env.FIREBASE_PROJECT;
-    delete process.env.CLOUD_RUNTIME_CONFIG;
+    delete process.env.PWD;
   });
 
-  it('loads config values from .runtimeconfig.json', () => {
-    mockRequire('../../../.runtimeconfig.json', { foo: 'bar', firebase: {} });
-    let loaded = config();
-    expect(loaded).to.not.have.property('firebase');
-    expect(loaded).to.have.property('foo', 'bar');
+  it('loads Firebase config from FIREBASE_CONFIG env variable', () => {
+    process.env.FIREBASE_CONFIG = JSON.stringify({
+      databaseURL: 'foo@firebaseio.com',
+    });
+
+    expect(firebaseConfig()).to.have.property(
+      'databaseURL',
+      'foo@firebaseio.com'
+    );
   });
 
   it('does not provide firebase config if .runtimeconfig.json not invalid', () => {
```

Comment:
I might be misunderstanding this test, but I think the description either should be "... if .runtimeconfig.json is invalid", or " .... if runtimeconfig.json is valid", depending on what the expected behavior is

Reply:
Hah didn't notice this was leftover from the previous code, nice catch.